get_DNAnexus_url.ps1

```diff
@@ -0,0 +1,29 @@
+# Works like wget, pass URLs to script to download them 
+Import-Module BitsTransfer
+Add-Type -AssemblyName System.IO.Compression.FileSystem
+# Function to unzip a file:
+function Unzip
+{
+    param([string]$zipfile, [string]$outpath)
+    [System.IO.Compression.ZipFile]::ExtractToDirectory($zipfile, $outpath)
+}
+# Each argument in the list of arguments:
+# Split the argument by comma (This is generated in the python script- porcess_duty_email.py)
+# First part of the argument is the URL path
+# Second part of the argument is the destination folder
+foreach ($url in $args) 
+{
+$array =$url.Split(",") 
+$path = $array[0]
+$output = $array[1]
+Write-Output "Downloading $path to $output"
+Start-BitsTransfer -Source $path -Destination $output
+# Check if the url contains the Results.zip file.
+# If yes then it will unzip the file and then delete the zipped file.
+if ($path.Contains("/Results.zip")) {
```

Comment:
FYI in the future this may need to change, as the results folder may be split based on dry lab - will raise an issue for a future release. Suggest something like if $path.endswith(".zip")
Also on line below Is  $filetoextract == $path?

Reply:
This can be changed in the future. For now, it works ok. 
Also on line below Is $filetoextract == $path? Yes.

Reply:
thanks, What I meant by  $filetoextract == $path? is that you can just use $path below - don't need to assign to a new variable.